config/passport.js

```diff
@@ -7,180 +7,184 @@ var mongoose = require('mongoose'),
       done(err, user);
     });
   });
 
-  //Use local strategy
-  passport.use(new LocalStrategy({
+  // Use local strategy
+  passport.use(new LocalStrategy(
+{
     usernameField: 'email',
     passwordField: 'password'
   },
-  function(email, password, done) {
-    User.findOne({
-      email: email
-    }, function(err, user) {
-      if (err) {
-        return done(err);
-      }
-      if (!user) {
-        return done(null, false, {
-          message: 'Unknown user'
-        });
-      }
-      if (!user.authenticate(password)) {
-        return done(null, false, {
-          message: 'Invalid password'
-        });
-      }
-      user.email = null;
-      user.hashed_password = null;
-      return done(null, user);
-    });
-  }
+    ((email, password, done) => {
+      User.findOne({
+        email
+      }, (err, user) => {
+        if (err) {
+          return done(err);
+        }
+        if (!user) {
+          return done(null, false, {
+            message: 'Unknown user'
+          });
+        }
+        if (!user.authenticate(password)) {
+          return done(null, false, {
+            message: 'Invalid password'
+          });
+        }
+        user.email = null;
+        user.hashed_password = null;
+        return done(null, user);
+      });
+    })
   ));
 
-  //Use twitter strategy
-  passport.use(new TwitterStrategy({
+  // Use twitter strategy
+  passport.use(new TwitterStrategy(
+{
     consumerKey: process.env.TWITTER_CONSUMER_KEY || config.twitter.clientID,
     consumerSecret: process.env.TWITTER_CONSUMER_SECRET || config.twitter.clientSecret,
     callbackURL: config.twitter.callbackURL
   },
-  function(token, tokenSecret, profile, done) {
-    User.findOne({
-      'twitter.id_str': profile.id
-    }, function(err, user) {
-      if (err) {
-        return done(err);
-      }
-      if (!user) {
-        user = new User({
-          name: profile.displayName,
-          username: profile.username,
-          provider: 'twitter',
-          twitter: profile._json
-        });
-        user.save(function(err) {
-          if (err) console.log(err);
+    ((token, tokenSecret, profile, done) => {
+      User.findOne({
+        'twitter.id_str': profile.id
+      }, (err, user) => {
+        if (err) {
+          return done(err);
+        }
+        if (!user) {
+          user = new User({
+            name: profile.displayName,
+            username: profile.username,
+            provider: 'twitter',
+            twitter: profile._json
+          });
+          user.save((err) => {
+            if (err) console.log(err);
+            return done(err, user);
+          });
+        } else {
           return done(err, user);
-        });
-      } else {
-        return done(err, user);
-      }
-    });
-  }
+        }
+      });
+    })
   ));
 
-  //Use facebook strategy
-  passport.use(new FacebookStrategy({
+  // Use facebook strategy
+  passport.use(new FacebookStrategy(
+{
```

Comment:
Use the previous comment to address the alignment and position of opening braces in the whole file